app/renderer/components/navigation/navigationBar.js

```diff
@@ -115,6 +125,77 @@ class NavigationBar extends React.Component {
       this.props.bookmarked
   }
 
+  // BEM Level: navigationBar__navigationButtonContainer
+  get stopButton () {
+    return <button className={cx({
+      normalizeButton: true,
+      [css(styles.navigationButton, styles.navigationButton_stopButton)]: true
+    })}
+      data-l10n-id='stopButton'
+      onClick={this.onStop}
+    />
+  }
+
+  // BEM Level: navigationBar__navigationButtonContainer
+  get reloadButton () {
+    return <LongPressButton className={cx({
```

Comment:
this just made me aware that browserButton should be a child component of longPress as well